backend/WebApi/Auth/AuthSetup.cs

```diff
@@ -21,11 +22,14 @@ public static void SetupLfAuth(IServiceCollection services, IConfiguration confi
         services.AddSingleton<JwtService>();
         services.AddAuthorization(options =>
         {
+            options.AddPolicy(nameof(ProjectAuthorizationRequirement), policy => policy.Requirements.Add(new ProjectAuthorizationRequirement()));
+
             //fallback policy is used when there's no auth attribute.
             //default policy is when there's no parameters specified on the auth attribute
             //this will make sure that all endpoints require auth unless they have the AllowAnonymous attribute
-            options.FallbackPolicy = options.DefaultPolicy;
+            options.FallbackPolicy = AuthorizationPolicy.Combine(options.DefaultPolicy, options.GetPolicy(nameof(ProjectAuthorizationRequirement)));
```

Comment:
So this modifies the fallback policy. But not the default, so if we were to put the authorize attribute somewhere then we would no longer be applying the project auth requirement without any trigger that this was happening.

Reply:
True. Then, I'll change the Default-Policy.